src/test/java/command/UserCommandTest.java

```diff
@@ -0,0 +1,45 @@
+package command;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertThrowsExactly;
+
+import org.junit.jupiter.api.Test;
+
+import exception.InvalidCommandException;
+
+public class UserCommandTest {
+    @Test
+    public void toCommand_exitCommand_success() {
+        assertInstanceOf(ExitCommand.class, UserCommand.toCommand("bye"));
+    }
+
+    @Test
+    public void toCommand_listCommand_success() {
+        assertInstanceOf(ListCommand.class, UserCommand.toCommand("list"));
+    }
+
+    @Test
```

Comment:
Perhaps the unit test can test if the resulting object possesses certain attributes rather than if its merely an instance of another